README.md

```diff
@@ -88,6 +91,9 @@ productTypeImport.importProductType(productType)
   // }
 })
 ```
+
+When there is an error during processing productTypes, process will by default save this error to `summary.errors` array and reject. If the configuration flag `importerConfig.continueOnProblems` is set to `true` the importer will only push error to summary and then continue with next productType.
```

Comment:
Something more along the lines of `process will by default save this error to "summary.errors" array and end`